hypermapper/random_scalarizations.py

```diff
@@ -413,10 +413,14 @@ def EI(
         scalarized_predictions = np.zeros(number_of_predictions)
         scalarized_value = 0
         for objective in regression_models:
+            if tmp_objective_limits[objective][1] - tmp_objective_limits[objective][0] == 0:
+               difference_obj = 10 ** -20
```

Comment:
If I'm not mistaken, the number we use here does not matter. Since the dividend of the division will be 0, resulting in `0/difference_obj = 0`. To make this clear, can you set `difference_obj = 1` and add a comment?